scripts/format-native.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+SCRIPT_DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)
+cd $SCRIPT_DIR/..
+
+LC_ALL=C
+
+# guess OS_TYPE if not provided
+if [ -z "$OS_TYPE" ]; then
+  case "$(uname -s | tr '[:upper:]' '[:lower:]')" in
+    cygwin_nt*|mingw*|msys_nt*)
+      OS_TYPE="windows"
+      ;;
+    linux*)
+      if [ "$(ldd /bin/ls | grep -m1 'musl')" ]; then
+        OS_TYPE="linux-musl"
+      else
+        OS_TYPE="linux-glibc"
+      fi
+      ;;
+    darwin*)
+      OS_TYPE="macos"
+      ;;
+  esac
+fi
+
+if [ "$OS_TYPE" == "windows"  ]; then
+    EXT=.exe
+else
+    EXT=
+fi
+
+# files to format
+NATIVE_FILES=$(find . -iname *.cpp -o -iname *.hpp -iname *.h -iname *.inc | grep -v ./packages | grep -v ./src/OpenTelemetry.AutoInstrumentation.Native/lib)
```

Comment:
This line is the main reason why it is not yet implemented in Nuke. I think we can create an issue to move the formatting logic to Nuke targets. Let me know what you think if we should address it ASAP or can we postpone it.